src/main/java/crawlercommons/sitemaps/AbstractSiteMap.java

```diff
@@ -53,6 +53,38 @@ protected DateFormat initialValue() {
         }
     };
 
+    private static final ThreadLocal<DateFormat> W3C_FULLDATE_FORMAT_WITH_OFFSET = new ThreadLocal<DateFormat>() {
+        protected DateFormat initialValue() {
+            SimpleDateFormat result = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ", Locale.ROOT);
+            result.setTimeZone(TimeZone.getTimeZone("UTC"));
+            return result;
+        }
+    };
+
+    protected static final ThreadLocal<DateFormat> RFC822_FULLDATE_FORMAT = new ThreadLocal<DateFormat>() {
+        protected DateFormat initialValue() {
+            return new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss Z", Locale.ROOT);
+        }
+    };
+
+    protected static final ThreadLocal<DateFormat> RFC822_NO_WEEKDAY_DATE_FORMAT = new ThreadLocal<DateFormat>() {
+        protected DateFormat initialValue() {
+            return new SimpleDateFormat("dd MMM yyyy HH:mm:ss Z", Locale.ROOT);
+        }
+    };
+
+    protected static final ThreadLocal<DateFormat> RFC822_SHORT_YEAR_DATE_FORMAT = new ThreadLocal<DateFormat>() {
+        protected DateFormat initialValue() {
+            return new SimpleDateFormat("EEE, dd MMM yy HH:mm:ss Z", Locale.ROOT);
+        }
+    };
+
+    protected static final ThreadLocal<DateFormat> RFC822_SHORTDATE_FORMAT = new ThreadLocal<DateFormat>() {
```

Comment:
It would be good to include a comment (before all of these date formats) for the RFC 822 standard, and why we've got extra options. E.g. if the various options are with/without the day of week, and yy vs. yyyy, and with or without the UTC timezone, then it seems like we need 8 unique combinations - so some explanation for why there are only 5.